examples/CLRWrapper/CLRWrapper.go

```diff
@@ -22,15 +23,14 @@ func main() {
 	}
 	fmt.Printf("[+] DLL Return Code: %d\n", ret)
 
-	
 	fmt.Println("[+] Executing EXE from memory")
-	exebytes, err := ioutil.ReadFile("helloworld.exe")
+	exebytes, err := ioutil.ReadFile(`hellworld.exe`)
```

Comment:
Can you fix this typo to `helloworld.exe`? Then it will work with my existing test exe 😄